package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-shopify",
-  "version": "16.0.1",
+  "version": "17.0.0-1",
```

Comment:
The trailing `-number` makes it a pre-release, which has lower precedence than a `17.0.0`.
Once we decide to release it, whomever is publishing should run `yarn version --new-version 17.0.0`.